### PR TITLE
Update Replayer's timestamp output

### PR DESCRIPTION
**Pull request by dfquaresma**
#### What is the purpose of this pull request?

This PR is related to #138 and updates the Replayer to output the timestamp in epoch format in nanoseconds. 
 
#### What problem is this solving?

See #138.

#### Types of changes
- [X] New feature (non-breaking change which adds functionality)
- [X] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: replayer/README.md [hunk 1]
```diff
@@ -45,17 +45,17 @@ The output is a _csv_ file with the following columns:
 - the HTTP method for this request
 - the response times, in nanoseconds
 - the HTTP status code for this request
-- the timestamp of the HTTP call
+- the timestamp of the HTTP call before calling, in epoc as nanoseconds
+- the timestamp of the HTTP call after calling, in epoc as nanoseconds
 
 A successful output to the input example used before would look like this:
 ```
 input_line,http_method,response_time,http_status,timestamp
```
Comment (paulofelipefeitosa) [inline replayer/README.md:53]: please change the csv header.

File: replayer/main.go [hunk 2]
```diff
@@ -67,7 +67,7 @@ func main() {
 
 	rawReqs := getRawRequests(inputData)
 	batches := loadBatches(rawReqs)
-	logger.Printf("input_line,http_method,response_time,http_status,timestamp\n")
+	logger.Printf("input_line,http_method,response_time,http_status,timestamp_before,timestamp_after\n")
```
Comment (paulofelipefeitosa) [inline replayer/main.go:70]: please add the suffix `nsec` to be easy to know the metric scale.

File: replayer/README.md (hunk 1 above)
Comment (dfquaresma) [reply replayer/README.md]: Thanks! It's done already.

File: replayer/main.go (hunk 2 above)
Comment (dfquaresma) [reply replayer/main.go]: Thanks! It's done already.

**Comment by dfquaresma** [comment]
Thank you @paulofelipefeitosa for review. Could you give another look?

**Pull request merged**